packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -14,6 +14,15 @@ export interface IRefreshSummaryResult {
 	isSummaryNewer: boolean;
 }
 
+export interface IStartSummaryResult {
+	/** The number of summarizerNodes at the start of the summary. */
+	nodes: number;
```

Comment:
Is there value in tracking this number? May be useful for % of invalid nodes, but not too sure (since any number of invalid nodes is bad)

Reply:
It may be useful to see if it's a partial node tree issue or a whole node tree issue. I'm not sure what I'll see, but I'd like to know if part of the tree is updated, or none of the tree is updated.